sql/src/main/java/io/crate/execution/engine/sort/BoundedSortingTopNCollector.java

```diff
@@ -55,8 +55,9 @@
     private final Comparator<Object[]> comparator;
     private final int offset;
     private final int maxSize;
-    private final RowAccounting rowAccounting;
+    private final RowAccounting<Object[]> rowAccounting;
 
+    private boolean accountForExtraSpare = true;
```

Comment:
I think this could be a local variable. Would reduce the scope and make it clearer where it is used.